src/coreclr/jit/jitconfigvalues.h

```diff
@@ -383,6 +383,15 @@ CONFIG_INTEGER(JitAggressiveInlining, W("JitAggressiveInlining"), 0) // Aggressi
 CONFIG_INTEGER(JitELTHookEnabled, W("JitELTHookEnabled"), 0)         // If 1, emit Enter/Leave/TailCall callbacks
 CONFIG_INTEGER(JitInlineSIMDMultiplier, W("JitInlineSIMDMultiplier"), 3)
 
+// Use a less aggressive and profile-independent inliner (which was the default one for < .NET 6.0).
+CONFIG_INTEGER(JitConservativeInliner, W("JitConservativeInliner"), 0)
+
+// Ex lclMAX_TRACKED constant.
+// Tested various sizes for max tracked locals. The largest value for which no throughput regression
+// could be measured was 512. Going to 1024 showed the first throughput regressions.
+// We anticipate the larger size will be needed to support better inlining.
```

Comment:
Would be good to redo these studies; a lot has changed since we last measured this.

Reply:
Yeah, will test TP impact of 1024.